bin/p2-dsctl/main.go

```diff
@@ -81,7 +81,7 @@ var (
 		This will output the hosts that match the selector,
 		The selector string uses same syntax as the kubernetes selectors without flags.
 		An example command is:
-		p2-dsctl test-selector "ice-cream in (the-fridge)"`,
+		p2-dsctl test-selector --selector "hardware in (lxc,metal)"`,
```

Comment:
depends who the example is for :( I could see this one going either way

We have helpful internal docs that describe valid selector types.